NextcloudTalk/SettingsTableViewController.swift

```diff
@@ -0,0 +1,663 @@
+            preferredStyle: .alert)
+        let okAction = UIAlertAction(title: NSLocalizedString("OK", comment: ""), style: .default, handler: nil)
+        errorDialog.addAction(okAction)
+        self.present(errorDialog, animated: true, completion: nil)
+    }
+
+    // MARK: Table view data source
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return getSettingsSections().count
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        let sections = getSettingsSections()
+        let settingsSection = sections[section]
+
+        switch settingsSection {
+        case SettingsSection.kSettingsSectionUserStatus.rawValue:
+            return 1
+        case SettingsSection.kSettingsSectionConfiguration.rawValue:
+            return getConfigurationSectionOptions().count
+        case SettingsSection.kSettingsSectionAbout.rawValue:
+            return AboutSection.kAboutSectionNumber.rawValue
+        case SettingsSection.kSettingsSectionAccounts.rawValue:
+            return NCDatabaseManager.sharedInstance().inactiveAccounts().count
+        default:
+            break
+        }
+        return 1
+    }
+
+    override func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        let sections = getSettingsSections()
+        let currentSection = sections[indexPath.section]
+        if currentSection == SettingsSection.kSettingsSectionUser.rawValue {
+            return 100
+        }
+        return 48
+    }
+
+    override func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        let sections = getSettingsSections()
+        let settingsSection = sections[section]
+
+        switch settingsSection {
+        case SettingsSection.kSettingsSectionUserStatus.rawValue:
+            return NSLocalizedString("Status", comment: "")
+        case SettingsSection.kSettingsSectionAccounts.rawValue:
+            return NSLocalizedString("Accounts", comment: "")
+        case SettingsSection.kSettingsSectionConfiguration.rawValue:
+            return NSLocalizedString("Configuration", comment: "")
+        case SettingsSection.kSettingsSectionAbout.rawValue:
+            return NSLocalizedString("About", comment: "")
+        default:
+            break
+        }
+        return nil
+    }
+
+    override func tableView(_ tableView: UITableView, titleForFooterInSection section: Int) -> String? {
+        let sections = getSettingsSections()
+        let settingsSection = sections[section]
+        if settingsSection == SettingsSection.kSettingsSectionAbout.rawValue {
+            let appName = (Bundle.main.object(forInfoDictionaryKey: "CFBundleDisplayName") as? String)!
+            let appVersion = (Bundle.main.object(forInfoDictionaryKey: "CFBundleShortVersionString") as? String)!
+            return appName + appVersion + copyright
+        }
+        return nil
```

Comment:
Missing some footers. Check the objective C settings implementation.

Reply:
footers added.